MovieQuiz-ios/MovieQuiz/Presentation/AlertPresenter.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  AlertPresenter.swift
+//  MovieQuiz
+//
+//  Created by Maxim Rustamov on 7.01.2023.
+//
+
+import UIKit
+
+struct AlertPresenter {
+    weak var delegate: AlertPresenterDelegate?
+    
+    func present(alert: AlertModel){
```

Comment:
Давай договоримся между скобками всегда оставлять один пробел
```suggestion
    func present(alert: AlertModel) {
```